### ISSUE TITLE
Not compatible with python3 - Arch Linux

### ISSUE THREAD

**Issue by chetwisniewski** [cdhowie/netflix-no-ipv6-dns-proxy#7]
Great tool, thanks so much for whipping this up. Just a note for users of systems that default to python3 like Arch Linux. The script gives a typing error "builtins.TypeError: a bytes-like object is required, not 'str'
" when run under python3.

To install the right package do: pacman -S python2-twisted
Then edit server.py to use python2 in the first line.

**Comment by cdhowie** [comment]
Can you give a full stack trace of that particular error?

**Comment by chetwisniewski** [comment]
I renamed it server3 for the python3 version. Here is the output, if you need something else please provide instructions:

```
./server3.py 
Unhandled Error
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/twisted/names/dns.py", line 2859, in datagramReceived
    self.controller.messageReceived(m, self, addr)
  File "/usr/lib/python3.5/site-packages/twisted/names/server.py", line 554, in messageReceived
    self.handleQuery(message, proto, address)
  File "/usr/lib/python3.5/site-packages/twisted/names/server.py", line 381, in handleQuery
    return self.resolver.query(query).addCallback(
  File "/usr/lib/python3.5/site-packages/twisted/names/common.py", line 73, in query
    return defer.maybeDeferred(method, query.name.name, timeout)
--- <exception caught here> ---
  File "/usr/lib/python3.5/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
    result = f(*args, **kw)
  File "/usr/lib/python3.5/site-packages/twisted/names/common.py", line 81, in lookupAddress
    return self._lookup(name, dns.IN, dns.A, timeout)
  File "/usr/lib/python3.5/site-packages/twisted/names/resolve.py", line 79, in _lookup
    d = self.resolvers[0].query(q, timeout)
  File "./server3.py", line 38, in query
    if self.__shouldBlock(query):
  File "./server3.py", line 30, in __shouldBlock
    domainParts = query.name.name and query.name.name.split('.')
builtins.TypeError: a bytes-like object is required, not 'str'
```

**Comment by cdhowie** [comment]
I'm assuming this should have been closed when #11 was merged.
